buildSrc/src/main/java/net/neoforged/neodev/NeoDevPlugin.java

```diff
@@ -70,26 +72,49 @@ public void apply(Project project) {
         // Task must run on sync to have MC resources available for IDEA nondelegated builds.
         NeoDevFacade.runTaskOnProjectSync(project, createSourceArtifacts);
 
+        // Obtain clean binary artifacts, needed to be able to generate ATs
```

Comment:
Most of the changes here were done to apply interface injection using JST. There's not really a need to bring them over in principle.

Reply:
Well with split sources we already have some delta. And the step numbers are messed up in this PR... Ok it doesn't matter, but it still irritates me 😅